tests/test_core_enhanced.py

```
@@ -0,0 +1,433 @@
+        assert 'target' not in X_test.columns
+
+    def test_stratified_split_classification(self):
+        """Test stratified split for classification."""
+        from core import split_data
+
+        data = pd.DataFrame({
+            'x': range(100),
+            'target': [0] * 80 + [1] * 20
+        })
+
+        X_train, X_test, y_train, y_test = split_data(
```

Ruff (0.14.14)

[warning] Unpacked variable `X_train` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)